label_studio/ml_models/migrations/0003_modelrun.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.23 on 2024-02-16 09:49
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('projects', '0026_auto_20231103_0020'),
+        ('organizations', '0006_alter_organizationmember_deleted_at'),
+        ('ml_models', '0002_thirdpartymodelversion'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ModelRun',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('project_subset', models.CharField(choices=[('All', 'All'), ('HasGT', 'HasGT')], default='HasGT', max_length=255)),
+                ('status', models.CharField(choices=[('Pending', 'Pending'), ('Completed', 'Completed'), ('Failed', 'Failed'), ('CANCELED', 'CANCELED')], default='Pending', max_length=255)),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='created at')),
+                ('triggered_at', models.DateTimeField(verbose_name='triggered at')),
```

Comment:
good call to add these, seems useful 🙂

Reply:
Also, I like the lack of "updated_at", since we really don't need it given the well-defined states a ModelRun can be in. This is much more useful